ccxt.d.ts

```diff
@@ -381,77 +381,75 @@ declare module 'ccxt' {
         encode (str: string): string;
-        decode (str: string): string;
-        account (): Balance;
-        commonCurrencyCode (currency: string): string;
-        market (symbol: string): Market;
-        marketId (symbol: string): string;
-        marketIds (symbols: string[]): string[];
-        symbol (symbol: string): string;
+        encodeURIComponent (...args: any[]): string;
         extractParams (str: string): string[];
-        createOrder (symbol: string, type: 'market'|'limit', side: 'buy'|'sell', amount: number, price?: number, params?: params): Promise<any>;
+        fetch (url: string, method?: string, headers?: any, body?: any): Promise<any>;
+        fetch2 (path: any, api?: string, method?: string, params?: params, headers?: any, body?: any): Promise<any>;
         fetchBalance (params?: params): Promise<Balances>;
-        fetchTotalBalance (params?: params): Promise<PartialBalances>;
-        fetchUsedBalance (params?: params): Promise<PartialBalances>;
+        fetchBidsAsks (symbols?: string[], params?: params): Promise<any>;
+        fetchClosedOrders (...args: any): Promise<any>; // TODO: add function signatures
+        fetchCurrencies (params?: params): Promise<Dictionary<Currency>>;
+        fetchDepositAddress (currency: string, params?: params): Promise<DepositAddressResponse>;
+        fetchDeposits (currency?: string, since?: number, limit?: number, params?: params): Promise<Transaction[]>;
         fetchFreeBalance (params?: params): Promise<PartialBalances>;
+        fetchFundingFees (...args: any): Promise<any>; // TODO: add function signatures
+        fetchL2OrderBook (...args: any): Promise<any>; // TODO: add function signatures
+        fetchLedger (...args: any): Promise<any>; // TODO: add function signatures
+        fetchMarkets (): Promise<Market[]>;
+        fetchMyTrades (symbol?: string, since?: any, limit?: any, params?: params): Promise<any>;
+        fetchOHLCV (symbol: string, timeframe?: string, since?: number, limit?: number, params?: params): Promise<OHLCV[]>;
+        fetchOpenOrders (symbol?: string, since?: number, limit?: number, params?: params): Promise<Order[]>;
         fetchOrder (id: string, symbol: string, params?: params): Promise<Order>;
         fetchOrderBook (symbol: string, limit?: number, params?: params): Promise<OrderBook>;
+        fetchOrderBooks (...args: any): Promise<any>; // TODO: add function signatures
+        fetchOrders (symbol?: string, since?: number, limit?: number, params?: params): Promise<Order[]>;
+        fetchOrderStatus (id: string, market: string): Promise<string>;
+        fetchStatus (...args: any): Promise<any>; // TODO: add function signatures
         fetchTicker (symbol: string, params?: params): Promise<Ticker>;
         fetchTickers (symbols?: string[], params?: params): Promise<Dictionary<Ticker>>;
+        fetchTime (...args: any): Promise<any>; // TODO: add function signatures
```

Comment:
Since you're here, what about changing this `Promise<any>` to `Promise<number>` ?

Reply:
Done.